### PR TITLE
Add instructions for running on Minishift

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@jpkrohling @pavolloffay If you could check the minishift instructions and possibly give them a try?

**Comment by pavolloffay** [comment]
I will do it, I haven't tried this metrics awesomeness :)

File: OpenShift.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+```
+minishift start --openshift-version=v3.6.0-alpha.2
+```
+
+You can use a more [recent version](https://github.com/openshift/origin/releases) if available.
+
+When fully started, then launch the dashboard using the link displayed following `The server is accessible via web console at:
+`. Navigate to the `New Project` overview page.
+
+NOTE: After using `oc create -f ...` to deploy something to Kubernetes, use the console to check that it
```
Comment (jkandasa) [inline OpenShift.md]: @objectiser Can I use the `developer` user here?

Comment (objectiser) [inline OpenShift.md]: Yes, in the openshift console, login as developer/developer - it is only the command line that needs to login as admin. I'll update the instructions.

File: OpenShift.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+it may be good to try having a single service monitor with a more generic label, which the services can
+be associated with.
+
+The next step is to create a route to make the Prometheus UI URL accessible:
+
+```
+oc create route edge prometheus --service=prometheus
+```
+
+Go to the OpenShift console Overview, expand the entry for `prometheus-prometheus` and you will see a link of the form `https://prometheus-myproject.192.168.42.161.nip.io/` which can be used to open the Prometheus console.
```
Comment (pavolloffay) [inline OpenShift.md]: It might be good to use link and not `links`

File: README.md [hunk 3]
```diff
@@ -9,6 +9,8 @@ First step is to install the OpenTracing compliant tracing system and Prometheus
 
 * [Kubernetes instructions](Kubernetes.md)
 
+* [OpenShift instructions](OpenShift.md)
```
Comment (pavolloffay) [inline README.md]: See line 8

> Currently only Kubernetes support is provided, although OpenShift will be added in the near future.

File: simple/OpenShift.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+```
+oc create route edge ordermgr --service=ordermgr
+```
+
+Final step is to obtain the URL for the new route from the OpenShift console, under the `ordermgr` application/deployment, of the form`https://ordermgr-myproject.192.168.42.161.nip.io/`, and use it to setup an environment variable `ORDERMGR`. NOTE: remove the final '/' when doing to the export, for example
+
+
+```
+export ORDERMGR=https://ordermgr-myproject.192.168.42.161.nip.io
```
Comment (pavolloffay) [inline simple/OpenShift.md]: this could use oc to get the url

Comment (objectiser) [inline simple/OpenShift.md]: Do you know the command to get the URL? I couldn't find anything obvious, except oc describe?

**Comment by objectiser** [comment]
@pavolloffay Thanks! Most changes applied - just question about the oc command for getting URL as I couldn't find anything equivalent to the minikube command. I've created a separate issue for potentially merging the minishift/kube instructions.

File: simple/OpenShift.md (hunk 4 above)
Comment (pavolloffay) [inline simple/OpenShift.md]: Kubectl supports jsonpath: 

https://kubernetes.io/docs/user-guide/jsonpath/
` oc get route jaeger-query -o=jsonpath="{.spec.host}" `

**Comment by objectiser** [comment]
@pavolloffay Thanks

**Comment by objectiser** [comment]
@jkandasa Do the updated instructions work ok for you?

**Comment by pavolloffay** [comment]
np, I hope that command works, I haven't tested it with curl.

Pavol Loffay
Cell SK:    +421 948 286 055
Email: p.loffay@gmail.com

File: simple/OpenShift.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+```
+oc create route edge ordermgr --service=ordermgr
+```
+
+Final step is to obtain the URL for the new route:
+
+
+```
+export ORDERMGR=`oc get route jaeger-query -o=jsonpath="{.spec.host}"`
```
Comment (pavolloffay) [inline simple/OpenShift.md]: It should be changed to order manager route/service. It was just an exaple

Comment (objectiser) [inline simple/OpenShift.md]: Woops, thanks :)

Comment (pavolloffay) [inline simple/OpenShift.md]: btw `-o=json` gives a nice hint how to select elements :)

**Comment by kevinearls** [comment]
I just worked through the example with no problems.  LGTM.

**Comment by objectiser** [comment]
@pavolloffay @kevinearls Thanks!

**Pull request merged**